public_examples/edgekv/postman/mock_edgekv_postman_environment.json

```diff
@@ -0,0 +1,114 @@
+{
+	"id": "e50ea4d2-ecaf-4dc6-9681-29079fb8702d",
+	"name": "MockEdgeKV",
+	"values": [
+		{
+			"key": "host",
+			"value": "",
+			"enabled": true
+		},
+		{
+			"key": "client_secret",
+			"value": "",
+			"enabled": true
+		},
+		{
+			"key": "access_token",
+			"value": "",
+			"enabled": true
+		},
+		{
+			"key": "client_token",
+			"value": "",
+			"enabled": true
+		},
+		{
+			"key": "network",
+			"value": "STAGING",
+			"enabled": true
+		},
+		{
+			"key": "accountId",
+			"value": "",
+			"enabled": true
+		},
+		{
+			"key": "contractId",
+			"value": "",
+			"enabled": true
+		},
+		{
+			"key": "groupId",
+			"value": "",
+			"enabled": true
+		},
+		{
+			"key": "productId",
+			"value": "",
+			"enabled": true
+		},
+		{
+			"key": "ewGroupId",
+			"value": "countries",
+			"enabled": true
+		},
+		{
+			"key": "resourceTierId",
+			"value": "",
+			"enabled": true
+		},
+		{
+			"key": "edgeWorkerId",
+			"value": "",
+			"enabled": true
+		},
+		{
+			"key": "cpCodeName",
+			"value": "",
+			"enabled": true
+		},
+		{
+			"key": "cpcode",
+			"value": "123456",
+			"enabled": true
+		},
+		{
+			"key": "namespaceId",
+			"value": "marketing",
+			"enabled": true
+		},
+		{
+			"key": "retentionInSeconds",
+			"value": 2592000,
+			"enabled": true
+		},
+		{
+			"key": "tokenName",
```

Comment:
I noticed that the token variables are not used in the generate token body, is that intentional?

Reply:
POST request revised to use variables